apps/qa/src/pages/ingest/ingest.py

```diff
@@ -2,20 +2,81 @@ def ingest():
     import streamlit as st
     from .helpers import archive_raw_data
     from pathlib import Path
+    import time
 
-    st.title("Ingest Dataset")
-
-    dataset_name = st.text_input("Dataset Name")
-    version = st.text_input("Version")
-    uploaded_file = st.file_uploader("Choose a file")
-    s3_path = Path("inbox") / dataset_name / version
+    if "ingest" not in st.session_state:
```

Comment:
Nice - I like having an `init` of the model/state.

Reply:
Though I don't think things like dataset_name, version, s3_path, or uploaded_file need to be in here - we really want to be capturing things that need to be tracked between resets that aren't handled by a simple input in `session_state`, but if we can get by with just the return value of an input we should stick with that